test/cogctl_test.exs

```diff
@@ -511,7 +513,7 @@ defmodule CogctlTest do
     NAME  ID
     """
 
-    run("cogctl relay-groups add myrelays my-test ")
+    run("cogctl relay-groups add myrelays --relays=my-test ")
```

Comment:
I'm not sure how this was working before. Though the test was written like the command should behave. I'm going to do a follow up PR to bring it inline with our design guidlines.